eth_account/hdaccount.py

```diff
@@ -0,0 +1,154 @@
+import os
+import hmac
+import hashlib
+import struct
+import codecs
+import ecdsa
+
+from hashlib import sha256
+from ecdsa.curves import SECP256k1
```

Comment:
We prefer to be flexible about what implementation people choose for this sort of thing.  We have a library called [`eth-keys`](https://github.com/ethereum/eth-keys) which implements the secp256k1 operations for public/private keys and signatures that I believe is going to be our preferred way to do this.

However, I'm not entirely sure it will do everything that you need for HD keys so I'm interested in your feedback on what is missing/needed if that is the case.

Reply:
i didn't know about that library. i will try it out ^^